docs/sources/reference/commandline/cli.md

```diff
@@ -1447,6 +1448,10 @@ Import to docker via pipe and `STDIN`.
 
     $ sudo tar -c . | docker import - exampleimagedir
 
+**Import from a local directory with local file:**
+
+    $ sudo docker import /local/path/to/exampleimage.tgz exampleimagedir
```

Comment:
Possibly I don't understand this correctly, but what is the purpose of `exampleimagedir` here?

Also "Import from a local directory with a local file" is confusing to me; what is the "directory"? Could it be just "Import from a local file"?